frontend/src/App.vue

```diff
@@ -34,17 +34,22 @@ html, body {
   --border-thick: 2px solid rgba(160, 178, 178, 0.5);
   --box-shadow-active: 0px 0px 0px 1px #ddd;
   --theme: #00a99d;
-  --theme-light: rgba(#00a99d, 0.6);
+  /*--theme-light: rgba(#00a99d, 0.6);*/
+  --theme-light: rgba(102, 203, 196);
   --font-large: 40px;
   --font-large-mobile: 24px;
   --font-small: 16px;
   --font-small-mobile: 12px;
 }
 
-main {
+#app {
```

Comment:
If you did * background-color: white and #app background-color: grey wouldn't it be more comprehensive rather the below subrules. Basically everything except for main should be white right?

Reply:
no cos you need spaces between stuff to show the background. See course page

Reply:
Not sure I fully understand what you mean.

Reply:
The spaces come from margins and everything that isn't covering the fallback main/app will be grey?